contribuer/pages/index.js

```diff
@@ -1,55 +1,619 @@
+            </li>
+          </ol>
+        </nav>
+
+        <section className="cta">
+          <h3>Outil de contribution</h3>
+          <p>Accédez à l’outil et commencez à contribuer.</p>
+          <p>
+            <a className="button" href="/admin/index.html">
+              Ouvrir l’outil de contribution
             </a>
-          </li>
-          <br />
-          <li>
+          </p>
+        </section>
+
+        <section aria-labelledby="guides">
+          <h3 id="guides">
             Tutoriel vidéo pour ajouter une aide dans l’outil de contribution ⬇️
-            <br />
-            <br />
-            <video width="600" controls preload="auto">
-              <source
-                src="https://betagouv.github.io/aides-jeunes-files/public/demo-outil-contribution.mp4"
-                type="video/mp4"
-              />
-              Your browser does not support the video tag.
-            </video>
-          </li>
-        </ul>
+          </h3>
+
+          <video controls preload="auto">
+            <source
+              src="https://betagouv.github.io/aides-jeunes-files/public/demo-outil-contribution.mp4"
+              type="video/mp4"
+            />
+            Your browser does not support the video tag.
+          </video>
+        </section>
+
+        <section aria-labelledby="contact">
+          <h3 id="contact">
+            😬 Vous avez une question ou une suggestion&nbsp;?
+          </h3>
+          <p>
+            N’hésitez pas à nous écrire pour nous les partager, cet outil
+            s’améliore au fur et à mesure que vous nous faites des
+            retours&nbsp;!
+          </p>
+        </section>
+
+        <section id="apropos" aria-labelledby="apropos-title">
+          <h2 id="apropos-title">01. À propos du simulateur</h2>
+
+          <h3>Pourquoi ce simulateur a‑t‑il été créé&nbsp;?</h3>
+          <p>
+            Les moins de 30 ans, qu’ils soient étudiant(e)s, demandeurs d’emploi
+            ou en décrochage scolaire, sont parmi les premières victimes de la
+            crise économique. Si des aides existent au niveau national ou local,
+            elles peuvent s’apparenter à un maquis difficile à naviguer pour le
+            public. Conséquence&nbsp;: le non‑recours aux aides de la part des
+            jeunes, par méconnaissance des dispositifs ou découragement. Ce
+            non‑recours, en cascade, a un impact sur la capacité des jeunes à
+            s’insérer dans la vie professionnelle ou à échapper à une situation
+            de pauvreté.
+          </p>
+          <p>
+            Bien que la cible prioritaire de ce simulateur soit les personnes de
+            moins de trente ans, il peut aussi être utilisé pour aider des
+            personnes de plus de trente ans à connaître leurs droits. En effet,
+            tous les dispositifs nationaux et locaux sont intégrés au simulateur
+            ou peuvent l’être, à condition qu’ils n’excluent pas les jeunes.
```

Comment:
suggestion :: uniformiser l'utilisation de "30" (ou "trente" ?) + éviter la répétition

```suggestion
            Bien que la cible prioritaire de ce simulateur soit les personnes de
            moins de 30 ans, il peut aussi être utilisé pour aider des
            personnes plus âgées à connaître leurs droits. En effet,
            tous les dispositifs nationaux et locaux sont intégrés au simulateur
            ou peuvent l’être, à condition qu’ils n’excluent pas les jeunes.
```